app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,221 @@
+---
+id: first_app.intellij-idea
+title: Building a Java application in IntelliJ IDEA
+slug: learn/intellij-idea
+type: tutorial
+category: start
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+more_learning:
+  youtube:
+    - H_XxH66lm3U
+    - 70_B2DyM8mU
+toc:
+- Installing IntelliJ IDEA
+- Creating a new project
+- Writing and editing code
+- Running your application
+- Testing
+- Debugging
+- Refactoring code
+- Documenting code
+- Searching and navigating
+- Version Control
+- Evolving the project
+- More information
+description: "IntelliJ IDEA is the most used Java IDE and helps you develop your Maven and Gradle Java projects faster."
```

Comment:
Is it necessary to talk about build tools here?
Also this description reads like an advertisement.

Reply:
Hi Ana-Maria, before I make additional changes to this PR, I just want to double check my understanding of your comment. Do you mean you want the content of this PR / article to be more like the content of the video? Or is this about which build system to use? Thanks!

Reply:
@ammbra As discussed: Change description to be more in line with the content (summary of the article)